src/common.h

```diff
@@ -31,6 +31,16 @@
 #include "maybe.h"
 #include "signal.h"  // IWYU pragma: keep
 
+// some versions of ncurses globally define TRUE/FALSE
```

Comment:
We shouldn't need to be doing this; please remove these defines and TRUE/FALSE definitions.